test/performance/perf-kinesis/src/main.rs

```diff
@@ -26,20 +26,13 @@
 /// stream, should not drift over time. (These measurements should become more concrete as
 /// we get more experience running this test).
 ///
-/// TODOs:
```

Comment:
This feels like an unnecessary duplicate of Github issues that's likely to fall out of date.